### DISCUSSION TITLE
Is the app still being developed?

### DISCUSSION THREAD

**Discussion by 3ter** [mozzius/graysky#514] [Q&A]
I came here because of the inline translation feature but then I saw that @mozzius is working at Bluesky now so I thought, probably it will be incorporated into the official app soon. But then I found  https://github.com/bluesky-social/social-app/issues/2863 and now I'm confused on why the official app can't do something that Graysky can - offer inline translations. Perhaps someone can help me understand 🙂?

**Comment by compuguy** [comment]
Honestly I'm not sure. The last update for the app on the Google play store was February 2025. So development might be dead for this app.. 😔
